packages/ketcher-react/src/script/ui/dialog/template/template-attach.jsx

```diff
@@ -226,13 +236,23 @@ class Attach extends Component {
 
     return (
       <TemplateEditDialog
-        title="Template edit"
+        title={this.mode === 'save' ? 'Save to Templates' : 'Template Edit'}
```

Comment:
1. Better to not write 'save' every time and create constant. 
If 'save' is used across multiple files create it in separate folder. If only in this file, create constant right here.

2. Better to move title to constant:

```
const title = this.mode === this.MODES.SAVE ? 'Save to Templates' : 'Template Edit'

<TemplateEditDialog
   title={title}
/>
```

Reply:
Thx! I agree